spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/MaskRCNN.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.models.maskrcnn
+
+import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.models.resnet.{Convolution, ResNet, ResNetMask, Sbn}
+import com.intel.analytics.bigdl.models.resnet.ResNet.{DatasetType, ShortcutType}
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.{T, Table}
+
+class MaskRCNN(val inChannels: Int,
```

Comment:
Move this class into `com.intel.analytics.bigdl.models.maskrcnn`

Reply:
oki, changed